Engine/ac/global_audio.cpp

```diff
@@ -354,24 +370,33 @@ void PlayMP3File (const char *filename) {
     int useChan = prepare_for_new_music ();
     bool doLoop = (play.music_repeat > 0);
 
-    if ((channels[useChan] = my_load_static_ogg(asset_name, 150, doLoop)) != NULL) {
-        channels[useChan]->play();
+    SOUNDCLIP *clip = my_load_static_ogg(asset_name, 150, doLoop);
```

Comment:
Nice. This was weird before as we'd overwrite the channel with null on failure.

Reply:
It still overwrites channel with null on failure, in the last else below, just explicitly.